ckan/setup/ckan.ini

```diff
@@ -42,6 +42,7 @@ beaker.session.timeout=900
 
 # CKAN caching
 ckan.cache_expires = 86400
```

Comment:
Caching pages for 24 hours seems agressive; do you think we could turn that down to maybe 1 hour?

Reply:
1 hour expiry will increase server load. Let us play safe and go with default value first. We can re-evaluate a good expiry after things are running on cloud.gov after a while.

Reply:
based on preliminary load tests on development space, it is safe to use 1 hour cache time instead of 1 day.